edit/secret.vue

```diff
@@ -52,13 +52,14 @@ export default {
 
       registryFQDN = Object.keys(auths)[0];
 
+      if (registryFQDN === 'index.dockerhub.io/v1/') {
```

Comment:
Don't think there should be a path here, registry creds are just the domain.

Reply:
I think that's what it should be for DockerHub: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/#create-a-secret-by-providing-credentials-on-the-command-line

Reply:
Okay that's weird because when you pull the image it doesn't contain that so the credential isn't actually a substring of the image URL.  But it appears you're right (though the variable isn't a FQDN then...).